src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,13 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        User user = new UserService().findByEmail(email);
+        if (user == null) {
+            return false;
+        }
+        String userEmail = user.getEmail();
+        String userPassword = user.getPassword();
+
+        return userEmail.equals(email) && userPassword.equals(password);
```

Comment:
```suggestion
        return userPassword.equals(password);
```
you checked it in `findByEmail` method